bleachbit/Action.py

```diff
@@ -408,15 +409,13 @@ def run_process():
                     rc = 0  # unknown because we don't wait
                     from subprocess import Popen
                     Popen(self.cmd)
-            except Exception, e:
+            except Exception as e:
                 raise RuntimeError(
                     'Exception in external command\nCommand: %s\nError: %s' % (self.cmd, str(e)))
             else:
                 if not 0 == rc:
-                    logger = logging.getLogger(__name__)
```

Comment:
According to [this SO answer](http://stackoverflow.com/questions/22807972/python-best-practice-in-terms-of-logging) it's best to use a logger instance per module. This way, we only need to change `getLogger(__name__)` to `getLogger('bleachbit.foo')` once per file.